senza/cli.py

```diff
@@ -116,6 +116,13 @@
 }
 
 
+def filter_output_columns(output_columns, filter_columns=list()):
```

Comment:
please use the singleton `None` as the default parameter, never an actual instance (does not make a difference here but is important in general as the default value is only instantiated once!)